src/components/Chat.vue

```diff
@@ -223,12 +223,15 @@ export default {
     },
     sendMessage() {
       const { id } = this.$route.params;
-      sessionsRef.child(`${this.key}/messages`).push({
-        type: "text",
-        data: this.input,
-        timeStamp: moment().valueOf()
-      });
-      this.input = "";
+      this.input = this.input.trim();
```

Comment:
Either assign this.input.trim() into a constant variable, or simply keep using this.input.trim()

Reply:
Done